.gitignore

```diff
@@ -23,4 +23,4 @@ stats.json
 .DS_Store
 Thumbs.db
 
-dashboard
+./dashboard
```

Comment:
can you also remove the generated binary from this commit?

Reply:
fixed